routing.rst

```diff
@@ -2640,6 +2648,10 @@ defined as annotations:
             ;
         };
 
+.. deprecated:: 6.4
+
+    Annotations are deprecated since Symfony 6.4, use attributes instead.
```

Comment:
Let's change the code snippets and the paragraph above to use the term "attribute(s)" instead of "annotation(s)" instead.